src/Entity/HomeBlock.php

```diff
@@ -158,83 +158,69 @@ public function getPosition(): ?int
         return $this->position;
     }
 
-    public function setPosition(int $position): self
+    public function setPosition(int $position): void
     {
         $this->position = $position;
-
-        return $this;
```

Comment:
Apparently, yes. I checked that there is no another method call of entity after.
Other suggestions why can it break anything ?